airbyte-workers/src/test/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowTest.java

```diff
@@ -1063,4 +1241,86 @@ private static void startWorkflowAndWaitUntilReady(final ConnectionManagerWorkfl
     }
   }
 
+  // TODO: bmoric -> use that in all test.
```

Comment:
it looks like this todo is already done

Reply:
I is done in the follow up PR, I will remove it there.